src/main/java/task.java

```diff
@@ -0,0 +1,49 @@
+public class task {
+    protected String name;
+    private Integer number;
+    private boolean isDone;
+    private String taskType;
+
+    public task(String name, Integer number, boolean isDone, String taskType){
+        setName(name);
+        setNumber(number);
+        setDone(isDone);
+        setTaskType(taskType);
+    }
+
+    public void setTaskType(String taskType){
+        this.taskType = taskType;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setNumber(Integer number) {
+        this.number = number;
+    }
+
+    public void setDone(boolean isDone) {
+        this.isDone = isDone;
+    }
+
+    public String getName(){
+        return name;
+    }
+
+    public Integer getNumber(){
+        return number;
+    }
+
+    public String getIsDone(){
+        if(isDone){
+            return "[✓]";
```

Comment:
Best to use \u???? instead of the symbol so it can be saved properly when testing